client/boot/app.js

```diff
@@ -49,3 +51,8 @@ window.AppBoot = () => {
 		user.once( 'change', () => boot( user ) );
 	}
 };
+
+// measure wasted component renders
+if ( process.env.NODE_ENV !== 'production' ) {
+	measureWastedRenders( React );
```

Comment:
not that we're going to merge this anyway, but wouldn't it be nice to only load this library from here instead of as an `import`?

Reply:
in order to prevent it from entering bundles?